packages/eslint-config-assignar-base/src/plugins/typescript.ts

```diff
@@ -11,9 +23,17 @@ const rules: Rules.Typescript = {
     // https://github.com/nzakas/eslint-plugin-typescript/blob/master/docs/rules/explicit-function-return-type.md
     'typescript/explicit-function-return-type': 'off',
 
-    // Leaving off accessibility modifier and making everything public can make your interface hard to use by others
-    // https://github.com/nzakas/eslint-plugin-typescript/blob/master/docs/rules/explicit-member-accessibility.md
-    'typescript/explicit-member-accessibility': 'error',
+    // Leaving off accessibility modifier and making everything public can make your interface hard to use by others.
+    // https://github.com/bradzacher/eslint-plugin-typescript/blob/master/docs/rules/explicit-member-accessibility.md
+    'typescript/explicit-member-accessibility': 'warn',
+
+    // Enforces naming of generic type variables.
+    // https://github.com/bradzacher/eslint-plugin-typescript/blob/master/docs/rules/generic-type-naming.md
+    'typescript/generic-type-naming': 'error',
+
+    // Enforce consistent indentation.
+    // https://github.com/bradzacher/eslint-plugin-typescript/blob/master/docs/rules/indent.md
+    'typescript/indent': 'off',
```

Comment:
I would consider switching this on instead of the base rule.
It handles more TS cases properly unlike the base rule.

Reply:
I should have merged and released https://github.com/bradzacher/eslint-plugin-typescript/pull/274 v soon - so it will do even better at handling indenting.